### PR TITLE
[IOTDB-771] Add audit log

### PR DESCRIPTION
**Pull request by qiaojialin**
Add audit log for connection and oprations.

Log file name: log_audit.log

### CODE REVIEW

File: server/src/main/java/org/apache/iotdb/db/service/TSServiceImpl.java [hunk 1]
```diff
@@ -1061,6 +1061,10 @@ public ServerProperties getProperties() {
   @Override
   public TSExecuteBatchStatementResp insertRecords(TSInsertRecordsReq req) {
     TSExecuteBatchStatementResp resp = new TSExecuteBatchStatementResp();
+    auditLogger
```
Comment (jixuan1989) [inline server/src/main/java/org/apache/iotdb/db/service/TSServiceImpl.java:1064]: better to add ifDebugEnabled

File: server/src/main/java/org/apache/iotdb/db/service/TSServiceImpl.java [hunk 2]
```diff
@@ -1337,6 +1345,8 @@ public TSExecuteBatchStatementResp createMultiTimeseries(TSCreateMultiTimeseries
       logger.info(INFO_NOT_LOGIN, IoTDBConstant.GLOBAL_DB_NAME);
       return RpcUtils.getTSBatchExecuteStatementResp(TSStatusCode.NOT_LOGIN_ERROR);
     }
+    auditLogger.info("Session-{} create multi timeseries, first is {}", currSessionId.get(),
```
Comment (jixuan1989) [inline server/src/main/java/org/apache/iotdb/db/service/TSServiceImpl.java:1348]: better to print how many time series

File: server/src/main/java/org/apache/iotdb/db/service/TSServiceImpl.java (hunk 1 above)
Comment (qiaojialin) [reply server/src/main/java/org/apache/iotdb/db/service/TSServiceImpl.java:1064]: fixed

File: server/src/main/java/org/apache/iotdb/db/service/TSServiceImpl.java (hunk 2 above)
Comment (qiaojialin) [reply server/src/main/java/org/apache/iotdb/db/service/TSServiceImpl.java:1348]: added

**Pull request merged**